modoboa/core/management/commands/load_initial_data.py

```diff
@@ -126,18 +129,18 @@ def handle(self, *args, **options):
             call_command(
                 "createapplication",
                 "--algorithm=RS256",
-                f"--redirect-uris={redirect_uri}",
+                f"--redirect-uris={redirect_uris}",
```

Comment:
Have you checked if it works well with more than one uri? (since you're using space a separator, I have some doubts...)

Reply:
Not tested tbf but shows like it in the [doc](https://github.com/jazzband/django-oauth-toolkit/blob/d862f531a2ab96fb3cf05f6ffbf9baf8192c1459/oauth2_provider/management/commands/createapplication.py#L35)